forwarder/update_handlers/ca_update_handler.py

```diff
@@ -72,16 +72,17 @@ def _monitor_callback(self, sub, response: ReadNotifyResponse):
                 self._cached_update is None
                 or response.metadata.status != self._cached_update.metadata.status
             ):
-                self._publish_message(
+                success = self._publish_message(
```

Comment:
Should similar changes be done in pva update handler as well?
https://github.com/ess-dmsc/forwarder/blob/6b14a1b3097e6526451a8f8b22faf869308f85da/forwarder/update_handlers/pva_update_handler.py#L96

Reply:
Thats a fair point. I will fix that.